external/loaders/loaders/batches/_batch.py

```diff
@@ -80,6 +80,8 @@ class BatchesFromMapperConfig(BatchesLoader):
     shuffle_samples: bool = False
     data_transforms: Optional[Sequence[Mapping]] = None
     ptop: float = vcm.calc.thermo.constants.TOA_PRESSURE
+    catalog_path: str = "/home/mr7417/ML_workflow/model_environment/fv3net/external \
+                         /vcm/vcm/catalog.yaml"
```

Comment:
This path will not exist on all platforms.  I recommend using the general path defined [here](https://github.com/ai2cm/fv3net/blob/81c29e865525233cd70e4ded8f67c1c06e3a7671/external/vcm/vcm/catalog.py#L5).
```suggestion
    catalog_path: str = vcm.catalog.catalog_path
```